Eatery/Controllers/Eateries/Campus/CampusEateryMealTableViewController.swift

```diff
@@ -88,80 +106,90 @@ class CampusEateryMealTableViewController: UITableViewController {
-        let stationTitles = sortedMenu.map { $0.0 }
-
-        // set title
-        let possibleTitle = stationTitles[indexPath.row]
-        if possibleTitle == "General" {
-            cell.titleLabel.text = ""
-            cell.titleCollapsed = true
-        } else {
-            cell.titleLabel.text = possibleTitle
-            cell.titleCollapsed = false
-        }
-
-        // set content
-        let menuItems = sortedMenu[indexPath.row].1
-        let names = menuItems.map { item -> NSMutableAttributedString in
-            if item.healthy {
-                return NSMutableAttributedString(string: "\(item.name.trim()) ")
-                    .appendImage(UIImage(named: "appleIcon")!, yOffset: -1.5)
-            } else {
-                return NSMutableAttributedString(string: item.name)
-            }
-        }
-
         let font = UIFont.systemFont(ofSize: 14)
         let paragraphStyle = NSMutableParagraphStyle()
         paragraphStyle.paragraphSpacing = 0.25 * font.lineHeight
-        let content = NSMutableAttributedString(
-            string: "",
+
+        let menuItem = sortedMenu[indexPath.section].1[indexPath.item-1]
+
+        var name: NSMutableAttributedString = NSMutableAttributedString(
+            string: "\(menuItem.name.trim()) ",
             attributes: [
                 .foregroundColor: UIColor.primary,
                 .font: font,
                 .paragraphStyle: paragraphStyle
             ]
         )
+        if menuItem.healthy {
+            name = name.appendImage(UIImage(named: "appleIcon")!, yOffset: -1.5)
+        }
+        cell.contentLabel.attributedText = name
+        cell.favorited = menuItem.favorite
+        return cell
+    }
 
-        if names.isEmpty {
-            content.append(NSMutableAttributedString(string: "No items to show"))
-        } else {
-            content.append(NSMutableAttributedString(string: "\n").join(names))
+    /// Create a table view cell when there are multiple dining stations in the menu
+    private func diningStationsHeaderCell(in tableView: UITableView, forRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(
+            withIdentifier: "MealStation",
+            for: indexPath
+        ) as! MealStationTableViewCell
+
+        guard let menu = menu else {
+            return emptyMenuCell(in: tableView, forRowAt: indexPath)
         }
 
-        cell.contentLabel.attributedText = content
+        let sortedMenu = Sort.sortMenu(menu.data.map { ($0, $1) })
+        let stationTitles = sortedMenu.map { $0.0 }
 
+        // set title
+        let possibleTitle = stationTitles[indexPath.section]
+        if possibleTitle == "General" {
+            cell.titleLabel.text = ""
+            cell.titleCollapsed = true
+        } else {
+            cell.titleLabel.text = possibleTitle
+            cell.titleCollapsed = false
+        }
         return cell
     }
-
+    /// Register favorites
```

Comment:
This shouldn't be a documentation comment imo. I would add it as a normal comment within the body, maybe. Also, there should be a newline between the closing bracket of one function and the definition/doc comment of the next one (i.e. newline after 187).